_posts/2020-02-14-running-a-great-socratic-seminar.md

```diff
@@ -0,0 +1,70 @@
+
+The format of the event itself is described here [LINK]. The question becomes: how can you successfully prepare for and operate a meetup that satisfies the requirements of a socratic seminar? 
+
+Presuming you already have a meetup page, the first step is finding event space. Depending on your locale, finding a host can be of varying difficulty. The main requirements for a location are that it is quiet, has ample seating and a projector; as your group grows in size you may also want some kind of microphone setup. Don't let the lack of microphone or projector deter you from all spaces as you can bring your own if you have no other choice.  
+
+If an obvious host is not immediately apparent, here are some connections you may want to seek out:
+- Where are other bitcoin/crypto meetups being held in your city? Attend those events and speak to the organizers and hosts, see if they are interested in hosting a socratic seminar. 
+- Ask your members--many may have an available space at their office or will have connections to space in your city.
+- Local universities. Reach out to any crypto/bitcoin clubs and see if they can book space on their campus.
+- Contact crypto/bitcoin companies in your city and see if they are interested in hosting.
+- Coworking spaces often allow members to book meeting rooms. 
+- Local libraries may have rooms you can book. 
+- Hacker spaces are often amenable to technically-focused discussion events.
+
+Once you have arranged a host, you'll want to promote the event. You only need a handful of people to host a socratic seminar so don't stress over having a massive turnout (especially in the beginning). If you schedule the events regularly (e.g. once a month) your attendance will trend upwards over time. Consistent scheduling is a key component to running a good meetup--become a staple of your local crypto community. A common concern surrounds the possibility of there being no enthusiasts in your local community. While theoretically possible, do not lean into that trap. If you can create a space for thoughtful and advanced discussion and you will likely be surprised by who shows up. 
+
+Because meetup.com is a watering hole for people interested in crypto, you can expect some natural inflow just by associating your meetup group to crypto-related keywords [LINK TO MEETUP FAQ]. Attending other crypto meetups and sharing your event with members of those groups is another easy way to increase your RSVP count. Don't be afraid of using various online communication channels to announce your event (e.g. Reddit, twitter, etc.) as well.
```

Comment:
twitter -> Twitter

Reply:
```suggestion
Because meetup.com is a watering hole for people interested in crypto, you can expect some natural inflow just by associating your meetup group to crypto-related keywords [LINK TO MEETUP FAQ]. Attending other crypto meetups and sharing your event with members of those groups is another easy way to increase your RSVP count. Don't be afraid of using various online communication channels to announce your event (e.g. Reddit, Twitter, etc.) as well.
```